### PR TITLE
Add support for the jetpack6.1 build

### PR DESCRIPTION
**Pull request by lanluo-nvidia**
# Description

Add support for the jetpack6.1 build.
Currently jetpack 6.1 has:
cuda: 12.6
python: 3.10
tensorrt: 10.3
DLFW: 24.08 (pytorch: 2.5.0)

It can build successfully on release/2.5 branch, will fail on the main branch for now, as main branch has used the pytorch method from 2.6.0 which is not in the DLFW24.08 distribution.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: toolchains/jp_workspaces/MODULE.bazel.jp61 [hunk 1]
```diff
@@ -0,0 +1,61 @@
+module(
```
Comment (narendasan) [inline toolchains/jp_workspaces/MODULE.bazel.jp61:1]: Make sure you indicate in the file name that this is a template

Comment (lanluo-nvidia) [reply toolchains/jp_workspaces/MODULE.bazel.jp61]: changed as suggested MODULE.bazel.tmpl.

**Comment by peri044** [comment]
1) script - move to docs
2) align with which pytorch to target (ngc or the stable ones )

**Comment by lanluo-nvidia** [comment]
this is just for testing in release/2.5 branch,
USE https://github.com/pytorch/TensorRT/pull/3211 instead for official merge to main.

**Pull request closed without merging**